optimum/intel/ipex/inference.py

```diff
@@ -19,6 +19,54 @@
 if is_ipex_available():
     import intel_extension_for_pytorch as ipex
 
+def sparse_model_config(model_config):
+    embedding_size = None
+    if hasattr(model_config, 'hidden_size'):

```

Comment:
You can use the [`NormalizedConfigManager`](https://github.com/huggingface/optimum/blob/main/optimum/utils/normalized_config.py#L151) to get the the model number of attention heads, hidden size and number of layer. See [here](https://github.com/huggingface/optimum/blob/main/optimum/onnxruntime/optimization.py#L62) and [here ](https://github.com/huggingface/optimum/blob/main/optimum/onnxruntime/optimization.py#L169-#L170),